src/server/game/Handlers/GroupHandler.cpp

```diff
@@ -779,7 +779,7 @@ void WorldSession::BuildPartyMemberStatsChangedPacket(Player* player, WorldPacke
     {
         uint64 auramask = player->GetAuraUpdateMaskForRaid();
         *data << uint64(auramask);
-        for (uint32 i = 0; i < MAX_AURAS; ++i)
+        for (uint32 i = 0; i < MAX_AURAS_GROUP_UPDATE; ++i)
```

Comment:
because of scope this variable will be destroyed when we end the loop, that's _probably_ the reason.